<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertWidget</class>
 <widget class="QWidget" name="insertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>数额</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="purposeInput">
     <item>
      <property name="text">
       <string>食品</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>住房</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>交通</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>日常百货</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>时装</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>娱乐</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>文化</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>水电</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>医药</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>电子产品</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>资金流通</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人情</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>其它</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>支付方式</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>用途</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="countInput"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QComboBox" name="payTypeInput">
     <item>
      <property name="text">
       <string>现金</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>微信</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>支付宝</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>信用卡</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="OkButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="remarksInput"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
